models/rule.go

```diff
@@ -0,0 +1,125 @@
+// pmm-managed
+// Copyright (C) 2017 Percona LLC
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+package models
+
+import (
+	"database/sql/driver"
+	"time"
+
+	"github.com/percona/pmm/api/managementpb"
+	iav1beta1 "github.com/percona/pmm/api/managementpb/ia"
+	"gopkg.in/reform.v1"
+)
+
+//go:generate reform
+
+// Rule represents alert rule configuration.
+//reform:ia_rules
+type Rule struct {
+	Template     *iav1beta1.Template    `reform:"template"`
```

Comment:
Don't use API types in database models – they can't be serialised correctly. That's why reform explodes